artifactory/services/promote.go

```diff
@@ -91,6 +93,7 @@ type BuildPromotionBody struct {
 	IncludeDependencies *bool  `json:"dependencies,omitempty"`
 	Copy                *bool  `json:"copy,omitempty"`
 	// FailFast options default is true. We need to avoid omitempty, otherwise, it would be forced to false if omitted.
+	// Notice that when running 'JFrog CLI V1' FailFast's default is false.
```

Comment:
jfrog-client-go is not supposed to be aware or includes information about the applications that use it. It is not jfrog-cli's client, but an independent library. I therefore suggest removign this line.